app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,4 @@
+#promote-search{
+  margin: 0px 10px 0px 5px;
```

Comment:
It doesn't look weird or anything without it. I was just going to liberal with my design. So I can just delete it out since it doesn't really look any different without it.